internal/mode/static/metrics/collectors/nginx.go

```diff
@@ -13,20 +16,50 @@ import (
 )
 
 const (
-	nginxStatusSock = "/var/run/nginx/nginx-status.sock"
-	nginxStatusURI  = "http://config-status/stub_status"
+	nginxStatusSock        = "/var/run/nginx/nginx-status.sock"
+	nginxStatusURI         = "http://config-status/stub_status"
+	nginxPlusAPISock       = "/var/run/nginx/nginx-plus-api.sock"
+	nginxPlusAPIURI        = "http://nginx-plus-api/api"
+	nginxStatusSockTimeout = 10 * time.Second
+)
+
+var (
+	upstreamServerVariableLabels               = []string{}
+	upstreamServerPeerVariableLabelNames       = []string{}
+	streamUpstreamServerVariableLabels         = []string{}
+	streamUpstreamServerPeerVariableLabelNames = []string{}
+	serverZoneVariableLabels                   = []string{}
+	streamServerZoneVariableLabels             = []string{}
 )
 
 // NewNginxMetricsCollector creates an NginxCollector which fetches stats from NGINX over a unix socket
 func NewNginxMetricsCollector(constLabels map[string]string) (prometheus.Collector, error) {
 	httpClient := getSocketClient(nginxStatusSock)
+
 	client, err := prometheusClient.NewNginxClient(&httpClient, nginxStatusURI)
 	if err != nil {
 		return nil, err
 	}
 	return nginxCollector.NewNginxCollector(client, metrics.Namespace, constLabels), nil
 }
 
+// NewNginxMetricsCollector creates an NginxCollector which fetches stats from NGINX over a unix socket
+func NewNginxPlusMetricsCollector(constLabels map[string]string) (prometheus.Collector, error) {
+	plusClient, err := createPlusClient()
+	if err != nil {
+		return nil, err
+	}
+	variableLabelNames := nginxCollector.NewVariableLabelNames(
```

Comment:
Since all the label names are empty slices, can we just pass `nginxCollector.VariableLabelNames{}` to `NewNginxPlusCollector`?